### PR TITLE
add support for sorting on id attribute

### PR DESCRIPTION
**Pull request by agos**

### CODE REVIEW

**Review by lucacorti — APPROVED** [review]

**Comment by agos** [comment]
@lucacorti I forgot to bump the version, re requesting a review

**Review by lucacorti — APPROVED** [review]

**Comment by agos** [comment]
@lucacorti you'll also have to merge it, without write access only you can run workflows and merge PRs

**Comment by lucacorti** [comment]
Yes I also need to release it to hex.pm, but you can run it if you depend in on GitHub master once merged.

**Pull request merged**
